sdks/go/pkg/beam/runners/prism/internal/worker/worker_test.go

```diff
@@ -18,7 +18,11 @@ package worker
 import (
 	"bytes"
 	"context"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/graph/coder"
```

Comment:
Something is weird about your import plugin. It starts a new group, in the middle of the standard lib imports...

Reply:
My bad, it doesn't start a new group, and it does sort them, it just puts them not with the rest of the non-std lib imports.

Reply:
Something is really weird. I'll just make sure to apply go fmt before PRs from now on and not rely on the IDE